.github/workflows/s3-deploy.yml

```diff
@@ -1,10 +1,8 @@
 name: Deploy to S3
 on:
-  push:
-    branches:
-      - v4
-    paths-ignore:
-      - package.json
+  release:
+    types:
+      - published
```

Comment:
This will be triggered by clicking the "Publish release" button of the release note, it is the same with the `publish.yml` which we use to execute `npm publish`.